R.NET/Function.cs

```diff
@@ -125,5 +125,30 @@ private SymbolicExpression createCallAndEvaluate(IntPtr argument)
               }
           }
       }
+
+      /// <summary>
+      /// Invoke the function with optionally named arguments by order.
+      /// </summary>
+      /// <param name="args">one or more tuples, conceptually a pairlist of arguments.
+      /// The argument names must be unique; null or empty string indicates unnamed argument. </param>
+      /// <returns>The result of the function evaluation</returns>
+      public SymbolicExpression InvokeNamedFast(params Tuple<string, SymbolicExpression>[] args)
+      {
+          IntPtr argument = Engine.NilValue.DangerousGetHandle();
+          var rfInstall = GetFunction<Rf_install>();
+          var rSetTag = GetFunction<R_SetExternalPtrTag>();

```

Comment:
In furthering my understanding of the R API, the external pointer is safe here because we don't reference the tags outside the scope of this function? I didn't quite grok the documentation I could find related to this function with RInternals.